data/libs/SpaceStation.lua

```diff
@@ -104,7 +104,7 @@ function SpaceStation:GetEquipmentPrice (e)
 	if equipmentPrice[self][e] then
 		return equipmentPrice[self][e]
 	end
-	local mul = e:IsValidSlot("cargo") and ((100 + Game.system:GetCommodityBasePriceAlterations(e)) / 100) or 1
+	local mul = e:IsValidSlot("cargo") and ((100 + Game.system:GetCommodityBasePriceAlterations(e.name)) / 100) or 1
```

Comment:
This comment is NOT to be addressed for this PR, and is something that could be solved in an unrelated, newcomer-friendly PR:

The `name` API for the equipment instances is misleading. I'd expect a name to be a string that is displayable on screen, whereas this value is more of an ID. It's just a small papercut in the many pitfalls of our codebase, but it's easily solvable.

Reply:
Absolutely! This is a 1am hack to get things working - I'm not sure about the purpose of the `volatile` sub-object, but I needed to be able to pass around commodity names (and this isn't meant to be the final iteration of commodities anyways, `name` just so happened to be an unused key to attach this datum to).